cli/flox-rust-sdk/src/models/env_registry.rs

```diff
@@ -191,6 +201,29 @@ impl RegistryEntry {
         }
         None
     }
+
+    /// Register an activation for an existing enviroment.
+    fn register_activation(&mut self, pid: ActivationPid) {
+        tracing::debug!("registering activation: {}", &pid);
```

Comment:
Nitpick: I'd prefer if we used structured logging here
```suggestion
        tracing::debug!(%pid, "registering activation");
```
The `%pid` syntax uses the `Display` impl for the type

Reply:
nit: you can also remove the `tracing` namespace. This file already imports `tracing::debug` and we are moving away from using `log::*`, so this should be unambigous..